DayPicker/DaySelectorDecorator.js

```diff
@@ -35,12 +35,17 @@ function generalizeSelected (selected, state) {
 	return selected.map(v => generalizeDay(v, state.firstDayOfWeek)).sort();
 }
 
-// Accepts a localized selected array and returns a "Sunday at index 0" array
+// Accepts a "Sunday at index 0" selected array or number and returns a
```

Comment:
Would you please explain why did you change this comment?
According to the first commit(https://github.com/enactjs/sandstone/pull/343/commits/a442055235a26bee21626b762776473ee5e2c04f), the input and output should not be changed.

Reply:
🤔 I think you are right. WDYT about the comment at line 28? Should we change that too?

Reply:
The comment on that line is also wrong. It describes precisely the opposite of the function's behavior.
The generalizeSelected function receives values that have already reflected the locale and convert them into values where Sunday is 0.
Comments could be changed to:
// Accept the localized array with "firstDayOfWeek" at index 0 and returns a "Sunday at index 0" array

Reply:
🆗